app/components/ButtonSecondary/index.tsx

```diff
@@ -0,0 +1,69 @@
+import {ReactNode} from 'react'
+import {Link} from '@remix-run/react'
+import './button_secondary.css'
+
+type ButtonProps = {
+  action?: string | (() => void)
+  children?: ReactNode
+  className?: string
+  tooltip?: string
+  active?: boolean
+  disabled?: boolean
+  props?: {[k: string]: any}
+}
+const ButtonSecondary = ({
+  children,
+  action,
+  tooltip,
+  className,
+  active = false,
+  disabled = false,
+  props,
+}: ButtonProps) => {
+  if (typeof action === 'string') {
+    return (
+      <Link
+        to={action}
+        className={
+          'button-secondary ' +
+          (active ? 'active' : disabled ? '' : 'inactive') +
+          (className ? ' ' + className : '')
+        }
+        onClick={(e) => {
+          if (disabled) {
+            e.preventDefault()
+          }
+        }}
+        {...props}
+      >
+        {children}
+        {tooltip && !disabled && <p className="tool-tip">{tooltip}</p>}
```

Comment:
from what I can see, this is the only major difference between this component and the `Button` one? That doesn't seem to merit a totally differenct component, as you can achieve the same by doing
```
<Button
   action={action}
   className={
          'button-secondary ' +
          (active ? 'active' : disabled ? '' : 'inactive') +
          (className ? ' ' + className : '')
        }
    active={active}
    disabled={disabled}
    {...props}>
       {children}
        {tooltip && !disabled && <p className="tool-tip">{tooltip}</p>}
</Button>
```
which in this case would be:
```
          <Button
            action={question.answerEditLink || ''}
            props={{target: '_blank', rel: 'noopener noreferrer'}}
        className="button-secondary"
          >
            <EditIcon className="no-fill" />
           <p className="tool-tip">Google Doc</p>
          </Button>
```

Reply:
Yes yes, you are correct. I've changed it, but note that I decided to put the tool tip element within the button component for consistency.